lib/stat_calculator.rb

```diff
@@ -35,4 +41,46 @@ def lowest_total_score
   def count_of_teams
     @teams.count
   end
+
+  # Calculates the winningest coach based on win percentage
+  def winningest_coach
```

Comment:
Winningest_coach and worst_coach should both take an argument of a season id like so:

Tests below were pulled directly from the spec_harness

it "#winningest_coach" do
    expect(@stat_tracker.winningest_coach("20132014")).to eq "Claude Julien"
    expect(@stat_tracker.winningest_coach("20142015")).to eq "Alain Vigneault"
  end

  it "#worst_coach" do
    expect(@stat_tracker.worst_coach("20132014")).to eq "Peter Laviolette"
    expect(@stat_tracker.worst_coach("20142015")).to eq("Craig MacTavish").or(eq("Ted Nolan"))
  end